content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/mpcie-4g-modem/content.md

```diff
@@ -358,6 +420,60 @@ udhcpc -q -f -n -i wwan0
 
 ![PRO 4G GNSS Module - Dynamic IP Configuration](assets/portentaMAXcarrier_mpcie_dynamic.png)
 
+### Modem Power Management
+
+The modems might become unresponsive, so it is recommended that power can be controlled through software to allow modem rebooting when necessary. The **gpioset** command should include a jumper and a 20 second delay for proper initialization.
```

Comment:
Explain that normally this is done automatically by our ModemManager script customization, such as

```
/usr/sbin/modem_on.sh
/usr/sbin/modem_off.sh
```

see systemctl cat ModemManager. If a user wants to handle the modem without ModemManager, he can use those scripts (much simplier). Inside those scripts there is the logic to distinguish between each carrier board.